velox/docs/functions/spark/array.rst

```diff
@@ -173,6 +173,14 @@ Array Functions
     if :doc:`spark.legacy_size_of_null <../../configs>` is set to false.
     Otherwise, returns -1 for null input.
 
+.. spark:function:: slice(array(E), start, length) -> array(E)
+
+    Returns a subarray starting at 1-based index ``start`` or from end if negative,
+    with ``length`` elements. Throws exception if ``start`` is 0. ::
```

Comment:
What happens if start points outside of the array?

Reply:
@boneanxs Would you document what happens when start or start + length point outside of the array and add examples? Let's also document what happens when length is negative or zero.